battle-service/src/main/java/com/company/battle/controllers/GameController.java

```diff
@@ -0,0 +1,68 @@
+package com.company.battle.controllers;
+
+import com.company.common.dtos.GameDto;
+import com.company.common.dtos.SettingsDto;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("api/v1/game")
+public class GameController {
+
+    @GetMapping("/{id}")
+    GameDto getGameById(@PathVariable UUID id) {
+        return null;
+    }
+
+    @GetMapping("/{link}")
+    GameDto getGameByLink(@PathVariable String link) {
+        return null;
+    }
+
+    @GetMapping
+    List<GameDto> getActiveGames() {
+        return null;
+    }
+
+    @PostMapping("/create")
+    GameDto createGame(@RequestBody GameDto gameDto) {
+        return null;
+    }
+
+    @PutMapping("/join")
+    GameDto joinGame(@RequestBody String link) {
+        return null;
+    }
+
+    @PutMapping("/leave")
+    GameDto leaveGame(@RequestBody GameDto gameDto) {
+        return null;
+    }
+
+    @GetMapping("/{id}")
+    SettingsDto getSettingsByGameId(@PathVariable UUID id) {
+        return null;
+    }
+
+    @PostMapping("/default")
```

Comment:
/game/default? What does it means?

Reply:
Idea was to show some default settings to user during game creation process. I will change method to get. And maybe I need return default GameDto or SettingsDto? I think return GameDto also make sense

Reply:
default settings could be hardcoded on client somewhere. 
don't need to fetch them from backend.

assume you don't need this method at all.